wikipedia-dark.user.css

```diff
@@ -384,11 +413,297 @@ regexp("https?:\/\/.*wik(i|t).*(org|jp).*") {
   .ve-ui-mwAceEditorWidget .ace_editor {
     border-color: var(--cm-6);
   }
+	/*Blyad's Fixes*/
+	.central-featured-lang :hover, .central-featured-lang :hover strong::before, .other-project-link:hover, .central-featured-lang :hover small::before, .central-featured-lang :hover bdi::before, .central-featured-lang :hover span::before {
+		background-color: #0000;
+		position: relative;
+	}
+	.central-featured-lang :hover::before, .other-project-link:hover::before {
+		content: '';
+		position: absolute;
+		width: 100%;
+		height: 100%;
+		left: 0;
+		top: 0;
+		margin: 0;
+		background-color: var(--blue-3);
+		opacity: .54;
+	}
+	.other-project-link div, .central-featured-lang small, .central-featured-lang strong {
+		filter: brightness(1);
+	}
+	.footer-sidebar-icon.sprite.svg-wikipedia_app_tile {
+		filter: invert(.835) hue-rotate(180deg);
+	}
+	span.jsl10n.sprite.svg-badge_ios_app_store, span.jsl10n.sprite.svg-badge_google_play_store, span.jsl10n.sprite.svg-badge_google_play_store:hover {
+		color: transparent;
+	}
+	[src="//upload.wikimedia.org/wikipedia/donate/a/ac/Close_oojs.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/a/a1/OOjs_UI_icon_caretDown.svg/20px-OOjs_UI_icon_caretDown.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/d/d8/Stylus-logo.svg/120px-Stylus-logo.svg.png"], [src="//upload.wikimedia.org/wikipedia/commons/thumb/4/4a/Dragon-149393.svg/20px-Dragon-149393.svg.png"] {
+		filter: invert(1);
+	}
+	[style="color:#0645ad; font-weight:bold;"] {
+		color: var(--assess-2) !important;
+	}
+	[style="font:3.6em/1.5 Gill Sans MT, GillSansSA, Calibri, sans-serif; color:#4a4a4a; padding:5px 0; -moz-box-shadow: 0px 5px 5px -6px #ccc; -webkit-box-shadow: 0px 5px 5px -6px #ccc; box-shadow: 0px 5px 5px -6px #ccc; vertical-align:middle;"] {
+		color: #bababa !important;
+	}
+	[style="color: DimGray; font-size:80%"] {
+		color: var(--gray-9a) !important;
+	}
+	.vector-user-menu-legacy #pt-anonuserpage {
+		color: #a4a9ad;
+	}
+	.vector-user-menu-legacy #pt-anonuserpage, .vector-user-menu-legacy #pt-userpage a {
+		background-image: none;
+		position: relative;
+	}
+	.vector-user-menu-legacy #pt-anonuserpage::before, .vector-user-menu-legacy #pt-userpage a::before {
+		content: url('https://pl.wikipedia.org/w/skins/Vector/resources/skins.vector.styles.legacy/images/user-avatar.svg?b7f58');
```

Comment:
Please include base64 images, linking to URL is not idea.

Reply:
Why? This isn't going to vanish and base64 takes more space which can result in slower page loading, if you could explain this, that would be nice.

Reply:
base64, will always display, its faster loading and you dont open unnecessary connections from the style to go read that file.

Obviously some optimized version of the png then converted/optimized to base64, the smaller the better.